molgenis-data-row-edit/src/main/frontend/package.json

```diff
@@ -13,9 +13,9 @@
     "build": "node build/build.js"
   },
   "dependencies": {
-    "@molgenis/molgenis-api-client": "^2.1.3",
+    "@molgenis/molgenis-api-client": "^3.1.0",
     "@molgenis/molgenis-i18n-js": "^0.1.1",
-    "@molgenis/molgenis-ui-form": "^1.1.0",
+    "@molgenis/molgenis-ui-form": "^1.2.0",
```

Comment:
If I remove the node_modules and run yarn install. My yarn.lock changes? Is this also occurring at your end?

Reply:
this may depend on the yarn version used
- what changes ?
- why would you want to remove your node modules folder

Reply:
It is the yarn version I think. I just upgraded to 1.10.1. I will downgrade to 1.6.0.